runtimes/rust/src/service.rs

```diff
@@ -97,15 +96,14 @@ pub trait Service: ServiceDispatcher + Debug + 'static {
     /// If you aren't interested in the processing of for the genesis block, you can use
     /// [`CallContext::in_genesis_block`] method and exit early if `true` is returned.
     ///
-    /// Also note that invocation of [`blockchain::Schema::height`] will **panic** if invoked within
-    /// `after_transactions` of the genesis block. If you are going to process the genesis
-    /// block and need to know current height, use [`blockchain::Schema::next_height`] to infer the
-    /// current blockchain height.
+    /// Also note that invocation of `exonum::blockchain::Schema::height` will **panic**
```

Comment:
Nit: "Also not that" looks redundant, and the `exonum::blockchain::Schema::height` is way too long. Are you against putting a link like `https://docs.rs/exonum/latest/exonum/blockchain/struct.Schema.html#method.height`?

Reply:
Yes, because links like this is too fragile.